app/src/main/java/com/sdpteam/connectout/event/location/LocationHelper.java

```diff
@@ -0,0 +1,62 @@
+package com.sdpteam.connectout.event.location;
+
+import static android.Manifest.permission.ACCESS_FINE_LOCATION;
+import static android.content.pm.PackageManager.PERMISSION_GRANTED;
+
+import android.app.Activity;
+import android.content.Context;
+import android.location.Location;
+
+import androidx.core.app.ActivityCompat;
+
+import com.google.android.gms.location.FusedLocationProviderClient;
+import com.google.android.gms.location.LocationServices;
+
+public class LocationHelper {
+
+    private static final int LOCATION_PERMISSION_REQUEST_CODE = 1;
+    private static LocationHelper INSTANCE;
+    private final FusedLocationProviderClient fusedLocationClient;
+    private Location lastLocation;
+
+    private LocationHelper(Context context) {
+        fusedLocationClient = LocationServices.getFusedLocationProviderClient(context);
+    }
```

Comment:
I am a bit perplex about this approach, since a user's true location does not change much and the user's location is unique, it would make more sense to have a static `lastLocation` and multiple instances of `LocationHelper`. What do you think ?